mitodl/bootcamp-ecommerce
# Use different URLs for receipt and cancel URLs

#### What are the relevant tickets?
Fixes #36 

#### What's this PR do?
Makes different urls by appending `?status={status}` to the receipt and cancellation URLs. CyberSource errors if the URLs are exactly the same

#### How should this be manually tested?
If you have the right CyberSource credentials you should be able to start a purchase and see the CyberSource page asking for your information
